ruledata.go

```diff
@@ -716,6 +688,38 @@ func SetRules(s *NetlinkConnection, content []byte) error {
 					}
 				}
 			}
+		case "file_rules":
+			vi := v.([]interface{})
+			for ruleNo := range vi {
+				rule := vi[ruleNo].(map[string]interface{})
+				path := rule["path"]
+				if path == "" {
+					log.Fatalln("Watch option needs a path!")
+				}
+				// TODO: Add key addition support
+				// key := rule["key"]
+				perms := rule["permission"]
+				log.Println("Setting File Permissions", path, perms)
+				var dd AuditRuleData
+				dd.Buf = make([]byte, 0)
+				add := AUDIT_FILTER_EXIT
+				action := AUDIT_ALWAYS
+				_audit_syscalladded = true
+
+				err := AuditSetupWatchName(&dd, path.(string))
```

Comment:
Oh, By libaudit I meant the `auditctl` code. 
Here are the functions I was talking about:
https://fedorahosted.org/audit/browser/trunk/src/auditctl.c#L270
https://fedorahosted.org/audit/browser/trunk/src/auditctl.c#L302
I was thinking that if any client code is dependent on `auditctl` type functions then we should be making that available to them.